test/service-test-utils/src/main/java/software/amazon/awssdk/testutils/service/AwsTestBase.java

```diff
@@ -93,16 +94,11 @@ public void describeTo(Description description) {
             }
 
             private void testNullOrEmpty(String value, String fieldName) {
```

Comment:
Minor nit: Changed string to use null and blank from empty or blank. Change the method name here?

Reply:
👍